_posts/2020-04-15-Cleard.md

```diff
@@ -0,0 +1,65 @@
+---
+author: John
+title: Week four co-winner Get Cleard, works to improve global screening processes
+---
+
+![Get Cleard](images/blog/Get Cleard.png)
+
+This week’s competition resulted in a tie-a first for our Codevid-19 global hackathon. Jaime Wood kindly offered to answer a few questions about her team’s entry, [Get Cleard](https://www.cleard.ai/).
+
+
```

Comment:
Only add one blank line between each paragraph/heading.  When working locally, our default formatter will remove these extra blank lines.

With you working on GitHub.com, I suggest following this rule, submitting the PR, then reviewing your work on the [deploy preview](https://deploy-preview-90--codevid-19-staging.netlify.app/2020/04/15/cleard) version of the post.  If it doesn't look ideal, I'll be the bearer of bad news and say that what you were intending to make happen (i.e. more padding between sections) is out of your hands.  I could modify the HTML/CSS for you, but there's slightly higher priority work right now.

Reply:
Fair enough.  I'll be moving local as absolutely quickly as possible-prob monday-i've got duelling courses this weekend (2 concurrent).  In the meantime, you bet I'll check in with netlify, a big improvement over github's preview btw.